### PR TITLE
add  p8 fields

### PR DESCRIPTION
**Pull request by arjun0419**
Adding required fields for adding p8s

### CODE REVIEW

**Review by jmadler — APPROVED** [review]

File: api.json [hunk 1]
```diff
@@ -125,6 +125,26 @@
             "description": "Safari: Password for safari_apns_p12 file",
             "writeOnly": true
           },
+            "apns_key_id": {
+              "type": "string",
+              "description": "iOS: Required if using p8. Unique identifier for the p8 authentication key.",
+              "nullable": true
```
Comment (kesheshyan) [inline api.json]: I think we don't need nullable anymore. All the properties in OpenAPI spec should be optional by default.

File: api.json [hunk 2]
```diff
@@ -125,6 +125,26 @@
...
+            },
+            "apns_team_id": {
+              "type": "string",
+              "description": "iOS: Required if using p8. Team ID generated by Apple for your developer account.",
+              "nullable": true
```
Comment (kesheshyan) [inline api.json]: ditto

File: api.json [hunk 3]
```diff
@@ -125,6 +125,26 @@
...
+            },
+            "apns_bundle_id": {
+              "type": "string",
+              "description": "iOS: Required if using p8. Bundle ID for your app in the Apple ecosystem.",
+              "nullable": true
```
Comment (kesheshyan) [inline api.json]: ditto

File: api.json [hunk 4]
```diff
@@ -125,6 +125,26 @@
...
+            },
+            "apns_p8": {
+              "type": "string",
+              "description": "iOS: Required if using p8. Base64 encoded p8 key",
+              "nullable": true
```
Comment (kesheshyan) [inline api.json]: ditto

**Review by kesheshyan — CHANGES_REQUESTED** [review]

File: api.json [hunk 5]
```diff
@@ -125,6 +125,26 @@
             "description": "Safari: Password for safari_apns_p12 file",
             "writeOnly": true
           },
+            "apns_key_id": {
```
Comment (kesheshyan) [inline api.json]: Seems like formatting is little bit off, could you please move these fields to the left?

File: api.json (hunk 1 above)
Comment (arjun0419) [reply api.json]: Fixed.

File: api.json (hunk 2 above)
Comment (arjun0419) [reply api.json]: Fixed.

File: api.json (hunk 3 above)
Comment (arjun0419) [reply api.json]: Fixed.

File: api.json (hunk 4 above)
Comment (arjun0419) [reply api.json]: Fixed.

**Review by kesheshyan — APPROVED** [review]

**Comment by kesheshyan** [comment]
Thanks for the help @arjun0419!

**Pull request merged**